PWGCF/EbyEFluctuations/Tasks/IdentifiedMeanPtFluctuations.cxx

```diff
@@ -212,15 +270,641 @@ struct IdentifiedMeanPtFluctuations {
+    histos.add("hdEdx_gen_pion", "hdEdx_gen_pion", kTH2D, {pAxis, dEdxAxis});
+
+    histos.add("hVar1x_gen", "hVar1x_gen", kTH2D, {subAxis, nchAxis});
+    histos.add("hVar2x_gen", "hVar2x_gen", kTH2D, {subAxis, nchAxis});
+    histos.add("hVarx_gen", "hVarx_gen", kTH2D, {subAxis, nchAxis});
+    histos.add("hVar2meanptx_gen", "hVar2meanptx_gen", kTH2D, {nchAxis, varAxis2});
+
+    histos.add("hVar1pix_gen", "hVar1pix_gen", kTH2D, {subAxis, nchAxis});
+    histos.add("hVar2pix_gen", "hVar2pix_gen", kTH2D, {subAxis, nchAxis});
+    histos.add("hVarpix_gen", "hVarpix_gen", kTH2D, {subAxis, nchAxis});
+    histos.add("hVar2meanptpix_gen", "hVar2meanptpix_gen", kTH2D, {nchAxis, varAxis2});
+
+    histos.add("hVar1kx_gen", "hVar1kx_gen", kTH2D, {subAxis, nchAxis});
+    histos.add("hVar2kx_gen", "hVar2kx_gen", kTH2D, {subAxis, nchAxis});
+    histos.add("hVarkx_gen", "hVarkx_gen", kTH2D, {subAxis, nchAxis});
+    histos.add("hVar2meanptkx_gen", "hVar2meanptkx_gen", kTH2D, {nchAxis, varAxis2});
+
+    histos.add("hVar1px_gen", "hVar1px_gen", kTH2D, {subAxis, nchAxis});
+    histos.add("hVar2px_gen", "hVar2px_gen", kTH2D, {subAxis, nchAxis});
+    histos.add("hVarpx_gen", "hVarpx_gen", kTH2D, {subAxis, nchAxis});
+    histos.add("hVar2meanptpx_gen", "hVar2meanptpx_gen", kTH2D, {nchAxis, varAxis2});
+
+    //========================MC Histograms Reconstructed=================================================
+
+    histos.add("hZvtx_after_sel_rec", "hZvtx_after_sel_rec", kTH1D, {vtxZAxis});
+    histos.add("hZvtx_after_sel8_rec", "hZvtx_after_sel8_rec", kTH1D, {vtxZAxis});
+
+    histos.add("ptHistogram_allcharge_rec", "ptHistogram_allcharge_rec", kTH1D, {ptAxis});
+    histos.add("ptHistogramPionrec", "ptHistogramPionrec", kTH1D, {ptAxis});
+    histos.add("ptHistogramKaonrec", "ptHistogramKaonrec", kTH1D, {ptAxis});
+    histos.add("ptHistogramProtonrec", "ptHistogramProtonrec", kTH1D, {ptAxis});
+
+    histos.add("ptHistogramPionrec_purity", "ptHistogramPionrec_purity", kTH1D, {ptAxis});
+    histos.add("ptHistogramKaonrec_purity", "ptHistogramKaonrec_purity", kTH1D, {ptAxis});
+    histos.add("ptHistogramProtonrec_purity", "ptHistogramProtonrec_purity", kTH1D, {ptAxis});
+
+    histos.add("ptHistogramPionrec_pdg", "ptHistogramPionrec_pdg", kTH1D, {ptAxis});
+    histos.add("ptHistogramKaonrec_pdg", "ptHistogramKaonrec_pdg", kTH1D, {ptAxis});
+    histos.add("ptHistogramProtonrec_pdg", "ptHistogramProtonrec_pdg", kTH1D, {ptAxis});
+
+    histos.add("Histogram_mass2_p_rec_beforesel", "Histogram_mass2_p_rec_beforesel", kTH1D, {ptAxis});
+    histos.add("Histogram_mass2_p_rec_aftersel", "Histogram_mass2_p_rec_aftersel", kTH1D, {ptAxis});
+  }
+
+  //++++++++++++++++++++++++Monte Carlo Reconstructed +++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+  void processMCReco(aod::MCCollisions::iterator const& mccoll, aod::MCTracks const& mcrectrack, aod::McParticles const& mcParticles, aod::McCollisions const& mcCollisions)
```

Comment:
In here it can be seen the apparent naming misleading effect: `MCCollisions` and `McCollisions`
By the way, why is the subscription to `McCollisions` needed? It is not used and the unused variable warning is going to be promoted to an error in the coming weeks